tommath.h

```diff
@@ -455,9 +455,9 @@ int mp_exptmod(mp_int *a, mp_int *b, mp_int *c, mp_int *d);
 
 /* number of primes */
 #ifdef MP_8BIT
-   #define PRIME_SIZE      31
+#  define PRIME_SIZE      31
```

Comment:
your PR title is lying :P

Reply:
I just wanted to check your review skills .....   ;-)